queries/robots/highlights.scm

```diff
@@ -0,0 +1,9 @@
+(comment) @comment @spell
+
+(directive) @constant
+
+(unknownDirective) @attribute
```

Comment:
I think that either both should be `@property`, or `(unknownDirective)` should not be highlighted by default.

Reply:
Done